http2/src/test/java/com/blazemeter/jmeter/http2/sampler/HTTP2StreamHandlerTest.java

```diff
@@ -24,183 +25,202 @@
+    http2StreamHandler.onData(stream, dataFrame, callback);
 
-        ByteBuffer b = ByteBuffer.wrap(data.getBytes());
+    resSR = http2StreamHandler.getHTTP2SampleResult();
 
-        Mockito.when(dataFrame.getData())
-                .thenReturn(b);
+    assertTrue(resSR.isSuccessful());
+  }
 
-        Mockito.when(dataFrame.isEndStream())
-                .thenReturn(true);
+  @Test
+  public void onResetTest() {
 
-        http2SampleResult = new HTTP2SampleResult();
+    Mockito.when(resetFrame.getError()).thenReturn(0x7);
```

Comment:
prefer using already defined constants instead of "magic numbers" (arbitrary numbers) to help better identify the meaning of the value. In this case for example instead of using 0x7 you can use ErrorCodes.REFUSED_STREAM_ERROR.code